workers/unity/Packages/com.improbable.gdk.tools/GdkToolsConfiguration.cs

```diff
@@ -13,9 +14,12 @@ public class GdkToolsConfiguration
         public List<string> SchemaSourceDirs = new List<string>();
         public string CodegenOutputDir;
         public string DescriptorOutputDir;
-        public string RuntimeIp;
         public string DevAuthTokenDir;
         public int DevAuthTokenLifetimeDays;
+        public bool SaveDevAuthTokenToFile;
+
+        internal string RuntimeIpEditorPrefKey = "RuntimeIp";
+        public string RuntimeIp => EditorPrefs.GetString(RuntimeIpEditorPrefKey);
```

Comment:
here we are using EditorPrefs instead of PlayerPrefs?

Reply:
yes because we'd expect the RuntimeIp to be system wide and not per project